.pre-commit-config.yaml

```diff
@@ -29,7 +29,7 @@ repos:
     hooks:
       - id: htmlhint
   - repo: https://github.com/pycqa/isort
-    rev: 5.10.1
+    rev: 5.12.0
```

Comment:
I had an issue with the pre-commit hook failing to run due to [this error](https://levelup.gitconnected.com/fix-runtimeerror-poetry-isort-5db7c67b60ff), so I updated the version to fix.